static/app/components/modals/projectCreationModal.tsx

```diff
@@ -152,7 +155,23 @@ export default function ProjectCreationModal({
       {step === 1 && (
         <Fragment>
           <Subtitle>{t('Set your alert frequency')}</Subtitle>
-          <IssueAlertOptions onChange={updatedData => setAlertRuleConfig(updatedData)} />
+          <IssueAlertOptions
+            alertSetting={
+              alertRuleConfig?.shouldCreateCustomRule
+                ? RuleAction.CUSTOMIZED_ALERTS
+                : alertRuleConfig?.shouldCreateRule
+                  ? RuleAction.DEFAULT_ALERT
+                  : RuleAction.CREATE_ALERT_LATER
+            }
+            interval={alertRuleConfig?.conditions?.[0]?.interval}
+            threshold={alertRuleConfig?.conditions?.[0]?.value}
+            metric={
+              alertRuleConfig?.conditions?.[0]?.id.endsWith('EventFrequencyCondition')
+                ? MetricValues.ERRORS
+                : MetricValues.USERS
+            }
+            onChange={updatedData => setAlertRuleConfig(updatedData)}
```

Comment:
ok this could be in a reusable function but lets try to merge this as it is and make further improvements in a follow-up. The PR is already large enough